packages/db/src/core/errors.ts

```diff
@@ -37,46 +37,14 @@ export const SHELL_QUERY_MISSING_ERROR = `${red(
 	'▶ Please provide a query to execute using the --query flag.'
 )}\n`;
 
-export const SEED_ERROR = (error: string) => {
-	return `${red(`Error while seeding database:`)}\n\n${error}`;
-};
-
 export const EXEC_ERROR = (error: string) => {
 	return `${red(`Error while executing file:`)}\n\n${error}`;
 };
 
-export const SEED_DEFAULT_EXPORT_ERROR = (fileName: string) => {
-	return SEED_ERROR(`Missing default function export in ${bold(fileName)}`);
-};
-
 export const EXEC_DEFAULT_EXPORT_ERROR = (fileName: string) => {
 	return EXEC_ERROR(`Missing default function export in ${bold(fileName)}`);
 };
 
-export const REFERENCE_DNE_ERROR = (columnName: string) => {
```

Comment:
Nit: maybe we can move `errors.ts` to the base of the `src` directory instead of splitting into two modules?

Reply:
We definitely can, it's interesting though that none of the errors are shared between. in which case, why have them together?

Reply:
Huh, honestly expected at least _one_ to be shared. Guess we can keep separate if that's the case!